roles/openshift_node/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 os_firewall_allow:
-- service: OpenShift kubelet
+- service: Kubernetes kubelet
```

Comment:
@detiber But you would have to take care of those cases too where firewalld might not be available and iptables is the only solution.